src/paths/ManagementPanel.tsx

```diff
@@ -114,54 +111,39 @@ function ManagementPanel() {
 
   const mapTiles = useMemo(
     () =>
-      tiles.map(
-        ({
-          id,
-          maxLat,
-          maxLon,
-          minLat,
-          minLon,
-          x,
-          y,
-          usersCount,
-          approvedByEditor,
-          assignedUserName,
-          gtfsStopsCount,
-          unconnectedGtfsStops,
-        }) => (
-          <>
-            <TextPath
-              positions={[
-                [maxLat, minLon],
-                [minLat, maxLon],
-              ]}
-              text={`${unconnectedGtfsStops}/${gtfsStopsCount}`}
-              color=""
-              orientation={-45}
-              center
-            />
-            <Rectangle
-              key={`map-${id}`}
-              bounds={[
-                [maxLat - 0.001, maxLon - 0.0015],
-                [minLat + 0.001, minLon + 0.0015],
-              ]}
-              pathOptions={{
-                color: getColor(id, usersCount || 0, approvedByEditor),
-              }}
-              eventHandlers={{
-                click: () => {
-                  handleTileSelected(id);
-                },
-              }}>
-              <Tooltip direction="top">
-                X: {x}, Y: {y} <br />
-                {t('managementPanel.assigned')}: {assignedUserName ? assignedUserName : '-'} <br />
-              </Tooltip>
-            </Rectangle>
-          </>
-        ),
-      ),
+      tiles.map(x => (
+        <>
+          <TextPath
+            positions={[
+              [x.maxLat, x.minLon],
```

Comment:
just comment: watch out for conflicts with pt-366

Reply:
Yes, I've fixed it in two branches to not avoid making a branch from PT-366.